.storybook/stories/DataFiltering/components/DataList.tsx

```diff
@@ -22,36 +22,39 @@ type DataListProps = {
 export const DataList = ({ loading, data }: DataListProps) => {
 	const { resetFilters } = useSortAndFilterContext();
 
-	if (loading) {
-		return (
-			<Stack as="ul" gap={1} id={tableId}>
-				{Array.from({ length: 10 }, (_, i) => i).map((i) => (
-					<DataListItemSkeleton key={i} />
-				))}
-			</Stack>
-		);
-	}
-
-	if (!data.length)
-		return (
-			<Stack gap={2} alignItems="flex-start" id={tableId} role="status">
-				<Stack gap={1}>
-					<HelpIcon size="lg" color="muted" />
-					<Heading type="h2" fontSize="lg">
-						No results found
-					</Heading>
-					<Text>Try adjusting your filter options.</Text>
+	return (
+		<div id={tableId}>
+			{loading ? (
+				<Stack as="ul" gap={1}>
+					{Array.from({ length: 10 }, (_, i) => i).map((i) => (
+						<DataListItemSkeleton key={i} />
+					))}
 				</Stack>
-				<Button onClick={resetFilters}>Clear filters</Button>
-			</Stack>
-		);
+			) : (
+				data.length > 0 && (
+					<Stack as="ul" gap={1}>
+						{data.map((item) => (
+							<DataListItem key={item.id} data={item} />
+						))}
+					</Stack>
+				)
+			)}
 
-	return (
-		<Stack as="ul" gap={1} id={tableId}>
-			{data.map((item) => (
-				<DataListItem key={item.id} data={item} />
-			))}
-		</Stack>
+			<Stack alignItems="flex-start" gap={2} role="alert">
```

Comment:
I don't think the "always rendered" for filter sidebar is necessary, but it gives us the best opportunity to make this pass the audit

Reply:
Why not necessary?

Reply:
I can't remember the first/exact comment I read about the idea that SRs 'only announce updates, not on mount' but here is one SO answer: https://stackoverflow.com/a/76856780

Reply:
Only because the other changes in this PR all announced fine with conditional rendering. I'd like to think that SRs have improved a bit since that recommendation was first made because everybody does conditional rendering

Reply:
Yeah, ok, I hear you. I was getting inconsistent announcements. It'd be great to get to the bottom of this at some point.